### DISCUSSION TITLE
[Idea] Add example of custom tab content for demoproductform2 example module

### DISCUSSION THREAD

**Discussion by Oksydan** [PrestaShop/PrestaShop#34938] [Ideas]
### The problem

It would be nice to have an example of how to add custom content to product page v2 custom tab feature.  Demoproductform2 module only shows how to add MoneyType in CustomTabType. This tab is likely to be used by many developers in the near future and an example of how to add a custom template to this tab could be very useful, since it is not so obvious how to do it correctly. 

### The solution

Add an example of how to add custom tab content template in the demoproductform2 example module.

### Do you plan to work on this subject?

- [X] I'm willing to contribute a [formal specification](https://github.com/PrestaShop/prestashop-specs).
- [ ] I'm willing to provide any wireframes or design assets required for this feature.
- [X] I'm willing to submit a Pull Request that implements this feature.
- [X] I'm willing to help verify that the implemented feature works as intended and produces no unintended side effects.

**Comment by kpodemski** [comment]
PR improving example module has just been merged.

https://github.com/PrestaShop/example-modules/pull/169
